akvo/akvo-rsr
# Show warning message for percentage indicators with updates

Show a warning message to users when a percentage indicator already has an
update which has been submitted or saved.

Closes #3501 

## Test plan:
- Create an update on a percentage indicator as one user. Preferably an admin so that they can be deleted.
- Verify that another user who has access to the project sees a warning message indicating that the indicator already has an update and a new update cannot be added. 

## Screenshots

![image](https://user-images.githubusercontent.com/315678/57924876-e4776180-7895-11e9-8825-fb5b7853c512.png)

![image](https://user-images.githubusercontent.com/315678/57924889-ec370600-7895-11e9-94c6-2e30fa2202c8.png)

![image](https://user-images.githubusercontent.com/315678/57924863-d7f30900-7895-11e9-89e0-27f3ad7c285d.png)
